static_bottom.inc

```diff
@@ -16,7 +16,7 @@
       </div>
       <div>
         <a href="https://www.snel.com/?utm_source=website&utm_medium=banner&utm_campaign=sponsored_banner&utm_id=grml-sponsored">
-        <img src="/sponsors/Logo-Snel.com.png" width="300" height="70" alt="snel logo"/>
+        <img src="/sponsors/Logo-Snel.com.png" width="300" height="100" alt="snel logo"/>
```

Comment:
JFTR: the logo has a native resolution of 750x280. The height should then actually be 112 to keep the same aspect ratio.

Reply:
I have corrected it to 300x112